clang/lib/Headers/opencl-c.h

```diff
@@ -17525,6 +17525,13 @@ void       __ovld __conv intel_sub_group_block_write_ui8( __global uint* p, uint
 
 #endif // defined(cl_intel_subgroups_char) || defined(cl_intel_subgroups_short) ||
        // defined(cl_intel_subgroups_long)
+
+#if defined(cl_intel_subgroup_buffer_prefetch)
+void __ovld __conv intel_sub_group_block_prefetch_ui(const __global uint *p);
+void __ovld __conv intel_sub_group_block_prefetch_ui2(const __global uint *p);
+void __ovld __conv intel_sub_group_block_prefetch_ui4(const __global uint *p);
+void __ovld __conv intel_sub_group_block_prefetch_ui8(const __global uint *p);
```

Comment:
minor NIT:
Can we match existing formatting?

```suggestion
void       __ovld __conv intel_sub_group_block_prefetch_ui(const __global uint *p);
void       __ovld __conv intel_sub_group_block_prefetch_ui2(const __global uint *p);
void       __ovld __conv intel_sub_group_block_prefetch_ui4(const __global uint *p);
void       __ovld __conv intel_sub_group_block_prefetch_ui8(const __global uint *p);
```